CSS/style.css

```diff
@@ -0,0 +1,640 @@
+}
+
+.spk-container-title {
+  align-self: center;
+  margin-top: 4%;
+  margin-bottom: 4%;
+}
+
+.spk-container-title::after {
+  content: "";
+  display: block;
+  width: 45px;
+  margin-left: 75px;
+  padding-top: 10px;
+  opacity: 0.7;
+  border-bottom: 3px solid #ec5242;
+}
+
+.spk-card-container {
+  align-self: center;
+  display: grid;
+  grid-template-columns: auto auto;
+  gap: 10%;
+  width: 90%;
+}
+
+.spk-card {
+  display: flex;
+  gap: 2%;
+}
+
+.spk-card-img {
+  max-width: 120px;
+  max-height: 140px;
+}
+
+.photo-sticker {
+  position: absolute;
+  z-index: -1;
+  margin-top: -28px;
+  margin-left: -27px;
+}
+
+.spk-card-content {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+}
+
+.spk-card-name {
+  margin: 0;
+  font-size: 20px;
+}
+
+.spk-card-title {
+  margin: 0;
+  color: #ec5242;
+  font-size: 15px;
+}
+
+.spk-card-title::after {
+  content: "";
+  display: block;
+  width: 30px;
+  padding-top: 10px;
+  opacity: 0.9;
+  border-bottom: 3px solid #d3d3d3;
+}
+
+.spk-card-text {
+  margin: 0;
+}
+
+.prtn-section-container {
+  background-color: #272a31;
+  color: #fff;
+  display: flex;
+  flex-direction: column;
+  text-align: center;
+  align-items: center;
+}
+
+.prtn-section-title {
+  font-weight: 100;
+  margin-top: 2%;
+  opacity: 0.5;
+}
+
+.prtn-section-title::after {
+  content: "";
+  display: block;
+  width: 35px;
+  margin-left: 20px;
+  padding-top: 20px;
+  opacity: 0.9;
+  border-bottom: 3px solid #ec5242;
+}
+
+.partners {
+  display: flex;
+  justify-content: space-around;
+  gap: 5%;
+  opacity: 0.4;
+  width: 70%;
+}
+
+.credits {
+  display: flex;
+  justify-content: center;
+  gap: 5%;
+  width: 100vw;
+  padding-top: 15px;
+  background-color: #e4e4e4;
+}
+
+.credits-brand {
+  display: flex;
+  max-width: 200px;
+}
+
+.footer-logo {
+  align-self: flex-start;
+  padding-top: 7px;
+}
+
+.footer-brand {
+  margin: 0;
+}
+
+.credits-description {
+  max-width: 400px;
+  margin: 0;
+}
+
+.footer-title {
+  margin: 0;
+}
+
+a {
+  text-decoration: none;
+  color: #000;
+}
+
+@media (max-width: 768px) {
+  .background {
+    background-size: cover;
+    top: 0;
+    height: 300px;
+  }
+
+  .background-gradient {
+    height: 300px;
+  }
+
+  .header-cont {
+    display: none;
+  }
+
+  #menu-button {
+    display: block;
+    cursor: pointer;
+    opacity: 0.8;
+    background-color: #eeecec;
+    padding: 5px 2px 2px 0;
+    border-radius: 5px;
+    margin: 15px 0 0 10px;
+  }
+
+  #logo {
+    display: none;
+  }
+
+  .nav-row {
+    margin: 0;
+    justify-content: unset;
+    background-color: transparent;
+  }
+
+  .menu-items-mobile {
+    display: none;
+    flex-direction: column;
+    gap: 15px;
+    width: 100vw;
+    text-align: center;
+    margin: 0;
+    padding-top: 10px;
+    padding-bottom: 10px;
+    background-color: #eeecec;
+  }
+
+  .button-container-mobile {
+    background-color: #eeecec;
+    width: 100vw;
+    padding: 0;
+    margin: 0;
+  }
```

Comment:
- Currently, when I open your hamburger menu, the contents of your website are scrolling just below it. 

Here is what I mean 👇🏽 

|Content of your menu and website |
| :---:   | 
|![menubad](https://user-images.githubusercontent.com/99246359/203666753-694e1210-5d33-4d6c-bb09-1e934a43f4e1.PNG)  |

- Why don't we refactor your code by adding `height: 100vh;` to your `.button-container-mobile` and `padding-top: 0;` to your `.nav-row`.

What do you think? 🤗

Reply:
Hi @Favourezeugwa  Thank you for the review, is this completely necessary? I have that desing (100 vh) on my portfolio hamb menu because my partner added it but I don't like it.  I could maybe remove the fixed position and leave it to just stay on the top, what do you think?

Reply:
Well, in my opinion, I think it will be much better for the menu to cover the entire screen height just for user experience. It didn't completely look nice, hence the feedback. It's an optional change now so you can implement it if you like.

But I would suggest you do it 🍒 🤗

Reply:
@Favourezeugwa I ended up following your advice, I tried to "decorate it" always following the design. Would you mind taking a look before I submit it for my last review?

Reply:
It looks pretty 🌸. I love the pink break lines 💟 
Sure @Gdiazdiaz, you can go ahead and submit for review. Everthing looks great!

For subsequent project, as long as you have implemented all required changes, you can always go ahead to submit 😄